app/src/screens/TabNavigator.tsx

```diff
@@ -78,7 +78,9 @@ const aboutTabOptions: BottomTabNavigationOptions = {
 
 const TabNavigator: React.FC = () => {
   const bannerProps = useConnectedActiveCallBanner();
-  const isAboutPageEnabled = Boolean(getEnvVariable('ENABLE_ABOUT_PAGE'));
+  const isAboutPageEnabled = React.useMemo(() => {
+    return Boolean(getEnvVariable('ENABLE_ABOUT_PAGE'));
```

Comment:
Unfortunately this won't work the way we hope it will.

```
Boolean('false') === true;
Boolean(null) === false;
Boolean(undefined) === false;
```

This is because `Boolean` will see that it's a `non-empty` string, not reading the contents of the string. Because it's an environment variable, we can reasonably assume it's always a string, but you can also check `typeof === 'string'` if you'd like. But we need to do `e === 'true'`.

Reply:
Oops my mistake